schema.ts

```diff
@@ -1244,6 +1268,23 @@ export function isArray(val?: Schema): val is ArraySchema {
   return Boolean(val && val.type === ValueType.Array);
 }
 
+/**
+ * Pulls out the item type of an Array schema, returning undefined if the Array contains another Array.
+ */
+export function maybeUnwrapArraySchema(
+  val?: Schema,
+): BooleanSchema | NumberSchema | StringSchema | GenericObjectSchema | undefined {
+  if (!isArray(val)) {
+    return val;
+  }
+
+  if (!isArray(val.items)) {
+    return val.items;
+  }
+
+  return undefined;
```

Comment:
Nit: Superfluous return statement - just remove it.

Reply:
Done